.github/workflows/ci_dockers.yml

```diff
@@ -62,64 +62,55 @@ jobs:
           push: false
         timeout-minutes: 50
 
-  build-cuda:
+  build-CUDA:
     runs-on: ubuntu-20.04
     strategy:
       fail-fast: false
       matrix:
         include:
           # todo: see notes in Dockerfile
-          - python_version: 3.6
-            pytorch_version: 1.4
-          - python_version: 3.7
-            pytorch_version: 1.7
-          - python_version: 3.8
-            pytorch_version: 1.8
-          # - python_version: 3.9
-          #   pytorch_version: 1.9
+          - python_version: "3.6"
+            pytorch_version: "1.5"
+          - python_version: "3.7"
+            pytorch_version: "1.6"
+          - python_version: "3.9"
+            pytorch_version: "1.9"
     steps:
       - name: Checkout
         uses: actions/checkout@v2
 
-      # for PT 1.4 we need to use CUDA 10.1
-      - run: |
-          cuda=$(python -c "print(10.2 if float(${{matrix.pytorch_version}}) >= 1.5 else 10.1)" 2>&1)
-          echo "::set-output name=CUDA::$cuda"
-        id: extend
-
       - name: Build CUDA Docker
         # publish master/release
         uses: docker/build-push-action@v2
         with:
           build-args: |
             PYTHON_VERSION=${{ matrix.python_version }}
             PYTORCH_VERSION=${{ matrix.pytorch_version }}
-            CUDA_VERSION=${{ steps.extend.outputs.CUDA }}
           file: dockers/base-cuda/Dockerfile
           push: false
         timeout-minutes: 50
 
-  build-conda:
+  build-Conda:
     runs-on: ubuntu-20.04
     strategy:
       fail-fast: false
       matrix:
         include:
-          - python_version: 3.6
-            pytorch_version: 1.4
-          - python_version: 3.7
-            pytorch_version: 1.7
-          - python_version: 3.8
-            pytorch_version: 1.8
-          - python_version: 3.9
-            pytorch_version: 1.9
+          - python_version: "3.6"
+            pytorch_version: "1.5"
+          - python_version: "3.7"
+            pytorch_version: "1.6"
+          - python_version: "3.8"
+            pytorch_version: "1.9"
+          - python_version: "3.9"
+            pytorch_version: "1.10"
```

Comment:
Just out of my curiosity, what's happening here? i see a few versions missing which I'm assuming is intentional

Reply:
the CI is testing randomly here, we don't test all 60 combinations